lib/views/reviews-view.js

```diff
@@ -537,7 +537,7 @@ export default class ReviewsView extends React.Component {
               <span className="github-Review-pendingBadge badge badge-warning">pending</span>
             )}
           </div>
-          <a className="github-Review-reportAbuseLink" title="report abuse" href="https://github.com/contact/report-content">
+          <a className="github-Review-reportAbuseLink" title="report abuse" href={`https://github.com/contact/report-content?report=${comment.author.login}&content_url=${encodeURIComponent(comment.url)}`}>
```

Comment:
thanks for tacking this, @vanessayuenn !

do we need to handle cases where `comment.author` doesn't exist?  I see some `comment : comment.author` kinds of ternaries sprinkled in other places.

Reply:
Good point! Yea I notice we have checks for `author` kinda inconsistently throughout `reviews-view`. I think this would be a good opportunity to clean that up.